tests/providers/aws/services/awslambda/awslambda_function_not_publicly_accessible/awslambda_function_not_publicly_accessible_test.py

```diff
@@ -134,21 +176,21 @@ def test_function_not_public(self):
             result = check.execute()
 
             assert len(result) == 1
-            assert result[0].region == AWS_REGION_US_EAST_1
+            assert result[0].region == AWS_REGION_EU_WEST_1
             assert result[0].resource_id == function_name
             assert result[0].resource_arn == function_arn
             assert result[0].status == "PASS"
             assert (
                 result[0].status_extended
                 == f"Lambda function {function_name} has a policy resource-based policy not public."
             )
-            assert result[0].resource_tags == []
+            assert result[0].resource_tags == [{}]
 
     def test_function_public_with_canonical(self):
```

Comment:
Why you don't use it here?

Reply:
I put in PR Description, I did not find a way to mock the CanonicalUser with moto.